specs/final-spec.md

```
@@ -0,0 +1,1010 @@
+| -------------------- | -------------------------------------------- |
+| Repository name      | `rms-link-checker`                           |
+| PyPI package name    | `rms-link-checker`                           |
+| CLI command name     | `link_check`                                 |
+| Language             | Python ≥ 3.10                                |
+| License              | (inherit from existing repo)                 |
+
+## 2. Installation
+
+### End-user
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)